testsuite/README.md

```diff
@@ -181,6 +187,6 @@ Thus, you don't need to use the latest local build of strimzi/kafka libraries to
 
 But if you want you can specify the kafka image to use for the test as follows:
 
-    mvn clean test -Dkafka.docker.image=strimzi/kafka:build-kafka-2.6.0  -f testsuite/keycloak-auth-tests
+    mvn clean test -Dkafka.docker.image=quay.io/strimzi/kafka:0.28.0-kafka-3.1.0 -f testsuite/keycloak-auth-tests
```

Comment:
Strimzi 0.29.0 supports Kafka 3.2.0. (just saying, no need to update it right here in this PR)

Reply:
Yeah, another PR to add a profile for 3.2.0 and make it the default.